### ISSUE TITLE
Inconsistence of "newline_between_rules" with @import or @media

### ISSUE THREAD

**Issue by ErickPetru** [beautify-web/js-beautify#769]
It seems that `"newline_between_rules": true` isn't working in the same way all the time, when there are imports or some nesting happens. Here's the original file:

```
@import url('https://fonts.googleapis.com/css?family=Roboto');
*, *:before, *:after { box-sizing: border-box; }
body { font: 400 1rem/1.4 'Roboto', Sans-serif; margin: 0; }
@media (min-width: 960px) {
    .cards { font-size: 0; }
    .cards .card { display: inline-block; font-size: 1rem; width: 50%; }
}
```

Here's the current result:

```
@import url('https://fonts.googleapis.com/css?family=Roboto');
*,
*:before,
*:after {
    box-sizing: border-box;
}

body {
    font: 400 1rem/1.4 'Roboto', Sans-serif;
    margin: 0;
}

@media (min-width: 960px) {
    .cards {
        font-size: 0;
    }
    .cards .card {
        display: inline-block;
        font-size: 1rem;
        width: 50%;
    }
}
```

And here's the desired result, keeping consistence of newlines on all cases:

```
@import url('https://fonts.googleapis.com/css?family=Roboto');

*,
*:before,
*:after {
    box-sizing: border-box;
}

body {
    font: 400 1rem/1.4 'Roboto', Sans-serif;
    margin: 0;
}

@media (min-width: 960px) {
    .cards {
        font-size: 0;
    }

    .cards .card {
        display: inline-block;
        font-size: 1rem;
        width: 50%;
    }
}
```

Am I missing something in my configuration or is it impossible?

**Comment by centigrade-thomas-becker** [comment]
I have the same problem using js-beautify 1.6.3

**Comment by MacKLess** [comment]
This appears to have been resolved with tests filed in v1.8.0-rc12. @ErickPetru  and @centigrade-thomas-becker  please feel free to double check at http://jsbeautifier.org/
